template.env

```diff
@@ -0,0 +1,64 @@
+# THIS IS A TEMPLATE .env FILE. 
+# MAKE A COPY OF THIS FILE NAMED .env, AND THEN MODIFY THE VALUES AS NEEDED.
+#
+# Tip: To verify your changes to the .env have taken effect, run the command: docker compose config
+# For more information on each environment variable, see the Neurobagel documentation.
+
+# ---- CONFIGURATION FOR graph ----
+# Replace ADMINPASSWORD with the secure password you want to set for the admin user
+NB_GRAPH_ADMIN_PASSWORD=ADMINPASSWORD
+# Replace DBUSER with the username you want to set for your graph database user
+NB_GRAPH_USERNAME=DBUSER
+# Replace DBPASSWORD with the secure password you want to set for the created database user
+NB_GRAPH_PASSWORD=DBPASSWORD
+# Replace my_db with the name you want to give your graph database
+NB_GRAPH_DB=repositories/my_db
+# Replace ./data with the path to your JSONLD files
+LOCAL_GRAPH_DATA=./data
+
+# Additional configurable parameters - uncomment to change the defaults
+# Change NB_GRAPH_PORT_HOST if port 7200 is already in use on the machine
+# NB_GRAPH_PORT_HOST=7200
+# NB_GRAPH_PORT=7200
+# NB_GRAPH_ROOT_HOST=~/graphdb-home
+# NB_GRAPH_ROOT_CONT=/opt/graphdb/home
+# NB_GRAPH_IMG=ontotext/graphdb:10.3.1
+# NB_GRAPH_ADDRESS=graph
+# ---------------------------------
+
+# ---- CONFIGURATION FOR n-API ----
+# Origins to allow requests from. To limit to specific origins, replace * with URLs separated by a space.
+NB_NAPI_ALLOWED_ORIGINS="*"
+# Whether to return only aggregate, dataset-level query results (excluding subject/session-level attributes).
+NB_RETURN_AGG=true
+
+# Additional configurable parameters - uncomment to change the defaults
+# Change NB_NAPI_PORT_HOST if port 8000 is already in use on the machine
+# NB_NAPI_PORT_HOST=8000
+# NB_NAPI_PORT=8000
+# NB_NAPI_TAG=latest
+# ---------------------------------
+
+# ---- CONFIGURATION FOR f-API ----
+# Additional configurable parameters - uncomment to change the defaults
+# Change NB_FAPI_PORT_HOST if port 8080 is already in use on the machine
+# NB_FAPI_PORT_HOST=8080
+# NB_FAPI_PORT=8000
+# NB_FAPI_TAG=latest
+# ---------------------------------
+
+# ---- CONFIGURATION FOR QUERY TOOL ----
+# Replace http://XX.XX.XX.XX with the URL (and port, if needed) of the Neurobagel API the query tool will talk to, *as the URL appears on a user's machine*, 
+# or localhost:<port> if the query tool will only be accessed from its host machine
```

Comment:
```suggestion
# You MUST replace http://XX.XX.XX.XX with the URL (and port, if needed) of the Neurobagel API the query tool will talk to
# Note: this URL value will determine where the query tool sends requests to when running on a USERS machine!
# So make sure you type the URL *as it would have to appear on a user's machine*,
# even if the query tool and the neurobagel API are served to the user from the same physical machine
# E.g.: https://neurobagel.myInstitute.edu/federation
# If you are testing your setup locally and will access the query tool on the SAME machine 
# that the Neurobagel API is hosted on, you can use localhost as the URL
# E.g. http://localhost:8000 (if API listens on port 8000)
```

This needs as much context as it can get - we want people to be able to start things immediately with minimal frustration!